pkg/normalize/validate.go

```diff
@@ -127,17 +113,12 @@ func checkLabel(label string, rType string, target, domain string, meta map[stri
 			return nil
 		}
 	}
-	// Don't warn for CNAMEs if the target ends with acm-validations.aws
-	// See https://github.com/StackExchange/dnscontrol/issues/519
-	if strings.HasPrefix(label, "_") && rType == "CNAME" && strings.HasSuffix(target, ".acm-validations.aws.") {
+	// Don't warn for records that start with _
+	// See https://github.com/StackExchange/dnscontrol/issues/829
+	if strings.HasPrefix(label, "_") {
```

Comment:
I'm getting warnings for `fd._domainkey.stackoverflow.com` and similar.  How about:

    if strings.HasPrefix(label, "_") || strings.Contains(label, "._") {

That should cover both cases.

Reply:
I merged your changes from #833 into #830.